tests/src/core/testqgsgraduatedsymbolrenderer.cpp

```diff
@@ -31,6 +35,7 @@ class TestQgsGraduatedSymbolRenderer: public QObject
     void cleanup();// will be called after every testfunction.
     void rangesOverlap();
     void rangesHaveGaps();
+    void _makeBreaksSymmetric(QList<double> &breaks, double symmetryPoint, bool astride);
```

Comment:
Shouldn't you call `_makeBreaksSymmetric` without arguments?

This is a list of function that the test will call

Reply:
no the 3 functions are linked, if I want to call it line 152, arguments are needed (compilator complains)